electronics/BMS_for_lithium_polymer_battery_pack.md

```diff
@@ -0,0 +1,39 @@
+# BMS for lithium polymer battery pack
+
+Battery monitoring is vital for electric vehicles, because the safety, operation, and the life of the battery depends on the battery system. This attribute is exactly the major function of the battery-management system (BMS)—to check and control the status of battery within their specified safe operating conditions. In this article, a BMS method is has been proposed using various circuits. 
+
+## BMS Building Blocks
+There are three main objectives common to all Battery Management Systems
+- Protect the cells or the battery from damage
+- Prolong the life of the battery
+- Maintain the battery in a state in which it can fulfill the functional 
+  requirements of the application for which it was specified.
+
+## Our approach:
+To design a threefold circuit:
+- over charge protection
+- discharge protection
+- battery level indication
+
+We decided to go the analogue rough for the BMS because we wanted to design a PCB and for current and temperature sensing we used an arduino simply because it is easy to logging data, we used lm 35 for temperature and acs712 for current sensing. In case the temperature exceeds the battery's manufacture specified operating temperature the battery will be disconnected from the output power.
+
+
```

Comment:
Remove empty lines.

Reply:
Removed. And done other changes as well. Please check.